### PR TITLE
Fix Diptera.54

### PR DESCRIPTION
**Pull request by adletaw**
This fixes the family for Diptera.54 and generates new data.

### CODE REVIEW

File: Rscripts/11_fixing_diptera54_family.R [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  n <- length(diptera_index) - 1
+  warning(paste("Found", n, "instances of Diptera.54.  Please check data for duplicate rows.",
+                sep = " "))
+}
+
+## Change the family
+new_df <- df
+new_df[diptera_index, "Family"] <- "Sciaridae"
+
+## Examine the differences (awesome feature Andrew!)
```
Comment (aammd) [inline Rscripts/11_fixing_diptera54_family.R]: SO COOL right!?

Comment (adletaw) [inline Rscripts/11_fixing_diptera54_family.R]: YES! :astonished:

File: Rscripts/11_fixing_diptera54_family.R [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+df <- read_data_safely("../data/10_Distributions_organisms_full.tsv")
+
+## Check for Diptera.54
+diptera_index <- which(df$nickname == "Diptera.54")
+
+if(length(diptera_index) - 1 > 0){
+  n <- length(diptera_index) - 1
+  warning(paste("Found", n, "instances of Diptera.54.  Please check data for duplicate rows.",
+                sep = " "))
+}
```
Comment (aammd) [inline Rscripts/11_fixing_diptera54_family.R]: PErhaps from L10 to here could be abstracted as a function?

Comment (adletaw) [inline Rscripts/11_fixing_diptera54_family.R]: Could do.  You think we will need to make changes like this again?

Comment (adletaw) [inline Rscripts/11_fixing_diptera54_family.R]: If so, I would be inclined to go all the way to L20 and make some function like `change_taxonomy` that does the whole thing.

**Pull request merged**